doc/tutorials/crypto.md

```diff
@@ -0,0 +1,106 @@
+# Cryptographic Methods in Elektra
+
+Elektra can protect the following aspects of your configuration:
+
+1. confidentiality, and
+2. integrity.
+
+Elektra provides two plugins to achieve this protection:
+
+1. `crypto`, and
+2. `fcrypt`.
+
+## Configuration File Encryption/Decryption
+
+The `fcrypt` plugin enables the encryption and decryption of entire configuration files, thus protecting the confidentiality of the configuration values.
+`fcrypt` utilizes GnuPG (GPG) for all cryptographic operations.
+The GPG key, which is used for encryption and decryption, is specified in the backend configuration under `encrypt/key`.
```

Comment:
That is too fast. It is not clear what the tutorial is about. What is a backend configuration? (maybe avoid the term altogether)